tests/sentry/sentry_metrics/test_postgres_indexer.py

```diff
@@ -30,3 +31,48 @@ def test_indexer(self):
         # test invalid values
         assert PGStringIndexer().resolve("beep") is None
         assert PGStringIndexer().reverse_resolve(1234) is None
+
+
+class PostgresIndexerV2Test(TestCase):
+    def setUp(self) -> None:
+        self.strings = {"hello", "hey", "hi"}
+        self.indexer = PGStringIndexerV2()
+
+    def tearDown(self) -> None:
+        for obj in StringIndexer.objects.all():
+            key = f"{obj.organization_id}:{obj.string}"
+            indexer_cache.delete(key)
+            indexer_cache.delete(obj.id)
+
+    def test_indexer(self):
+        org_id = self.organization.id
+        org_strings = {org_id: self.strings}
```

Comment:
please pass two org_ids. There is a lot of code that assemble keys between org id and string, for many organizations at the same time. If you test one org and one string only that misses a lot of corner cases